### DISCUSSION TITLE
M2/M3 Compatibility

### DISCUSSION THREAD

**Discussion by Sagider** [asb2m10/dexed#396] [Q&A]
Is the Dexed plugin compatible with Apple M2/M3, in the change-logs they've mentioned compatibility with Apple M1, however, nothing about M2/M3 processors.

**Comment by Beamwork** [comment]
Unfortunately not on my mini m2 pro..Hopes are up for a upgrade of this fine emulation!

**Reply by asb2m10** [reply]
Current Dexed version is compatible with any Apple ARM64 chips, that includes M2 and M3. If you have an issue it is probably elsewhere. What DAW are you using ?

**Reply by Beamwork** [reply]
Hi! Thanks for reaching out!

I´m om Logic 11.0.1. 
[Dexed Crashed validation report.pdf](https://github.com/user-attachments/files/16674037/Dexed.Crashed.validation.report.pdf)

MacOs 14.6.1 (23G93)
Mini M2 Pro

**Reply by asb2m10** [reply]
This is a known issue with Logic 11 and Dexed 0.9.7 :/ Please test with this dev builds https://github.com/asb2m10/dexed/actions/runs/10239126765

Otherwise 0.9.8 should be released in the coming weeks.

**Comment by Beamwork** [comment]
Says the file is damaged and that I should put it in the trash. I will wait
for the 0.9.8 update.

Thanks again Pascal!

tir. 27. aug. 2024 kl. 07:00 skrev Pascal Gauthier ***@***.***
>:

> This is a known issue with Logic 11 and Dexed 0.9.7 :/ Please test with
> this dev builds https://github.com/asb2m10/dexed/actions/runs/10239126765
>
> Otherwise 0.9.8 should be released in the coming weeks.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/asb2m10/dexed/discussions/396#discussioncomment-10458666>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKTA7OKUBOZMT5NQKKIJTN3ZTQBXTAVCNFSM6AAAAABMVNPTOKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBVHA3DMNQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Vennlig hilsen
Nils-Arne Øvergård
Tlf.nr. 970 88 313

**Comment by asb2m10** [comment]
Since official release of 0.9.8 everything should run out the box for Logic (and M2/M3 compatibility).

**Reply by Beamwork** [reply]
Thank you for your work!
